bridge-ext/src/main/java/com/liferay/faces/bridge/ext/renderkit/html_basic/internal/ResourceRendererLiferayImpl.java

```diff
@@ -64,15 +70,37 @@ public ResourceRendererLiferayImpl() {
 		// Defer initialization of wrappedRenderer until restoreState(FacesContext, Object) is called.
 	}
 
-	public ResourceRendererLiferayImpl(Renderer renderer, String componentFamily, String rendererType,
-		boolean renderHeadResourceIds) {
+	public ResourceRendererLiferayImpl(Renderer wrappedRenderer, String componentFamily, String rendererType,
+		boolean renderHeadResourceIds, String primeFacesCSSDefaultDataSennaTrackValue) {
 
-		this.wrappedRenderer = renderer;
+		this.wrappedRenderer = wrappedRenderer;
 		this.componentFamily = componentFamily;
+		this.primeFacesCSSDefaultDataSennaTrackValue = primeFacesCSSDefaultDataSennaTrackValue;
 		this.rendererType = rendererType;
 		this.renderHeadResourceIds = renderHeadResourceIds;
 	}
 
+	/* package-private */ static NameValuePair<String, Object> getDataSennaTrack(UIComponent componentResource, boolean isCSS,
```

Comment:
@stiemannkj1 The `is` prefix is typically limited to boolean getter methods, not parameter names. Should be `css` or `styleSheet` instead of `isCss`